tests/lib/util/source-code.js

```diff
@@ -1794,6 +1810,30 @@ describe("SourceCode", () => {
             assert.notProperty(node.parent, "parent");
         });
 
+        it("should use visitorKeys", () => {
+            const text = "a + b";
+            const ast = espree.parse(text, DEFAULT_CONFIG);
+            let node;
+
+            // default
+            sourceCode = new SourceCode(text, ast);
+            node = sourceCode.getNodeByRangeIndex(0);
+            assert.strictEqual(node.type, "Identifier");
+
+            // no traverse BinaryExpression#left
```

Comment:
Could this test be split in two?

Reply:
Actually, the former is just a check to make sure that `visitorKeys` option changed the result. I replaced that with a comment.